app/assets/stylesheets/pregnancies.scss

```diff
@@ -16,8 +16,12 @@ $font-size: 17px;
   }
 }
 
-#menu {
-  border-right: 1px solid black;
+#sections {
+  border-left: 1px solid black;
+  padding-left: 15%;
+}
+
+#patient_detail {
```

Comment:
Yeah, since I'm adding the `#menu` div back in, I will change this back to `#menu` as well